app/src/main/java/io/pslab/activity/MapsActivity.java

```diff
@@ -58,7 +58,7 @@ private void addMarkers(JSONArray markers) throws JSONException {
         for (int i = 0; i < markers.length(); i++) {
             JSONObject marker = markers.getJSONObject(i);
             m.setPosition(new GeoPoint(marker.getDouble("lat"), marker.getDouble("lon")));
-            m.setTitle(marker.getString("data") + " @ " + marker.getString("date"));
+            m.setTitle("Data recorded on" + marker.getString("date"));
```

Comment:
Please use string resources instead of hardcoded text. Refer to checklist point 1 in PR template.